src/main/java/com/homeadvisor/kafdrop/model/TopicVO.java

```diff
@@ -30,119 +30,125 @@
    // partition state
    // delete supported?
 
-
    public TopicVO(String name)
    {
-      this.name = name;
+	  this.name = name;
    }
 
    public String getName()
    {
-      return name;
+	  return name;
    }
 
    public void setName(String name)
    {
-      this.name = name;
+	  this.name = name;
    }
 
    public Map<String, Object> getConfig()
    {
-      return config;
+	  return config;
    }
 
    public void setConfig(Map<String, Object> config)
    {
-      this.config = config;
+	  this.config = config;
+   }
+
+   public Map<Integer, TopicPartitionVO> getPartitionMap()
+   {
+	  return partitions;
```

Comment:
If we're going to expose this map, let's make it unmodifiable (e.g. `Collections.unmodifiableMap(partitions)`)